library/Icingadb/Common/LoadMore.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+/* Icinga DB Web | (c) 2021 Icinga GmbH | GPLv2 */
+
+namespace Icinga\Module\Icingadb\Common;
+
+use Icinga\Module\Icingadb\Widget\ItemList\PageSeparatorItem;
+use Icinga\Module\Icingadb\Widget\ShowMore;
+use ipl\Orm\ResultSet;
+use ipl\Web\Url;
+
+trait LoadMore
+{
+    /** @var int */
+    protected $pageSize;
+
+    /** @var int */
+    protected $pageNumber;
+
+    /** @var Url */
+    protected $loadMoreUrl;
+
+    /**
+     * Set the page size
+     *
+     * @param int $size
+     *
+     * @return $this
+     */
+    public function setPageSize($size)
+    {
+        $this->pageSize = $size;
+
+        return $this;
+    }
+
+    /**
+     * Set the page number
+     *
+     * @param int $number
+     *
+     * @return $this
+     */
+    public function setPageNumber($number)
+    {
+        $this->pageNumber = $number;
+
+        return $this;
+    }
+
+    /**
+     * Set the url for LoadMore widget
```

Comment:
`LoadMore` doesn't exist as widget, at least I read this as if there's a `Icingadb\Widget\LoadMore`. Just describe what the url is used for: to fetch *more* items